build.gradle

```diff
@@ -128,6 +128,9 @@ rat {
     "**/_config.yml",
     "**/_config_test.yml",
     "**/.htaccess",
+    "website/.bundle/**/*",
```

Comment:
Shouldn't we add these to .gitignore instead of here?

Not that this wouldn't be needed if the output from the website project went underneath `website/build/...`